app/helpers/photo_helper.rb

```diff
@@ -0,0 +1,88 @@
+module PhotoHelper
+
+  def resize_photo photo, params
+    rover_name = photo.rover.name.downcase
+    suffix_data = lookup_suffix rover_name, params[:size]
+
+    if !suffix_data.nil?
```

Comment:
It tends to be easier to read flow control by using positive statements. For example, here it would be easier for most people to read `if suffix_data.present?` or even `if suffix_data`, since `nil` is falsey.

Reply:
Thanks for catching this! I fixed it to be a positive logical statement

Reply:
As far as I can see this is still a negative statement. It's not a big problem, but generally people find positive statements easier to follow.

Reply:
Oh, looks like it was changed on line 38, but not here.